OrbitGl/TrackAccessibility.cpp

```diff
@@ -0,0 +1,120 @@
+// Copyright (c) 2020 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "TrackAccessibility.h"
+
+#include "GlCanvas.h"
+#include "TimeGraph.h"
+#include "Track.h"
+
+namespace orbit_gl {
+
+using orbit_accessibility::AccessibleInterface;
+
+const AccessibleInterface* AccessibleTrackContent::AccessibleParent() const {
+  return track_->AccessibilityInterface();
+}
+
+std::string AccessibleTrackContent::AccessibleName() const {
+  CHECK(track_ != nullptr);
+  return track_->GetName() + "_content";
+}
+
+AccessibilityRect AccessibleTrackContent::AccessibleLocalRect() const {
+  CHECK(track_ != nullptr);
+  CHECK(track_->GetTimeGraph() != nullptr);
+
+  auto& layout = track_->GetTimeGraph()->GetLayout();
+  return AccessibilityRect(0, static_cast<int>(layout.GetTrackTabHeight()),
+                           static_cast<int>(track_->GetSize()[0]),
+                           static_cast<int>(track_->GetSize()[1]));
+}
+
+AccessibilityState AccessibleTrackContent::AccessibleState() const {
+  AccessibilityState result;
+  return result;
+}
```

Comment:
Should this always return a dummy? Maybe use "*static* AccessibilityState result;" ?

Reply:
It will also fix compilation I think.

Reply:
True, I changed the way `AccessibilityState` is defined during this PR, this made sense before :-) Changed, thanks.